debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -80,6 +81,19 @@
                     + "'initial' (the default) to specify the connector should run a snapshot only when no offsets are available for the logical server name; "
                     + "'initial_schema_only' to specify the connector should run a snapshot of the schema when no offsets are available for the logical server name. ");
 
+    public static final Field TABLENAME_CASE_INSENSITIVE = Field.create("database.tablename.case.insensitive")
+        .withDisplayName("Oracle table name case insensitve mode")
+        .withType(Type.BOOLEAN)
+        .withDefault(false)
+        .withImportance(Importance.LOW)
+        .withDescription("Oracle table node case sensitve true or false, use true mode when you need to work with oracle 11g.");
+    
+    public static final Field POS_VERSION = Field.create("database.position.version")
```

Comment:
Maybe we could merge both new fields into a one - called `OracleVersion`. Thet would be enum  that will be configured with values `11g` and `12c` and will provide setting for table name insensitive handling and pos version?